src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -145,54 +145,115 @@ public static class PropertyNames
             }
         }
 
+        /// <summary>
+        ///     Specifies the configuration property(ies) that provide the basic authentication credentials.
+        /// </summary>
+        [Obsolete("This property will be depreciated in a future version of this library in favor of the BasicAuthCredentialsSource property")]
```

Comment:
Here and everywhere else: did you mean "deprecated"?

Reply:
i did indeed

Reply:
actually, I mean removed...